spec/factories/users.rb

```diff
@@ -6,6 +6,7 @@
     duns_number { Faker::Company.duns_number }
     name { Faker::Name.name }
     email
+    github_login 'ghlogin'
```

Comment:
why did we add it here? If there is not a validation but we require it for something to work, we should add the validation